graphql2/graphqlapp/rotation.go

```diff
@@ -375,3 +375,36 @@ func (m *Mutation) UpdateRotation(ctx context.Context, input graphql2.UpdateRota
 	}
 	return true, nil
 }
+
+func (a *Query) NextRotationHandoffTimes(ctx context.Context, input *graphql2.NextRotationHandoffTimesInput) ([]time.Time, error) {
+	var result []time.Time
+	var err error
+
+	err = validate.Many(
+		err,
+		validate.Range("input.count", input.Count, 0, 20),
+		validate.Range("input.hours", input.Hours, 0, 99999),
+	)
+	if err != nil {
+		return result, err
+	}
+
+	loc, err := util.LoadLocation(input.TimeZone)
+	if err != nil {
+		return result, errors.Wrap(err, "failed to load location for "+input.TimeZone)
+	}
+
+	rot := &rotation.Rotation{
+		Start:       input.Start.In(loc),
+		ShiftLength: input.Hours,
+		Type:        rotation.TypeHourly,
+	}
+
+	t := time.Now()
+	for len(result) < input.Count {
+		t = rot.EndTime(t)
+		result = append(result, t)
+	}
+
+	return result, err
```

Comment:
Use `return result, nil` here instead since the err has already been checked, for clairity.